contribs/github-bot/internal/config/config.go

```diff
@@ -55,10 +56,14 @@ func Config(gh *client.GitHub) ([]AutomaticCheck, []ManualCheck) {
 			Then:        r.Never(),
 		},
 		{
-			Description: "Pending initial approval by a review team member (and label matches review triage state)",
+			Description: "Pending initial approval by a review team member, or review from tech-staff",
```

Comment:
A review team member or an org member? (It may be the same thing; I just wanted to check that it wasn't an error)

Reply:
the rule is a bit "wider" than the description, but essentially it's because org members who are not tech staff are considered "on par" with the review team.

ie., while Leon is not in the review team, if he takes a look at a PR and approves it, it should be considered as if it were approved by an actual review team member.